tests/Feature/SharingTestScenariosAbstract.php

```diff
@@ -275,13 +281,13 @@ public function testTwoPhotosInPublicAlbum(): void
 		$responseForAlbum->assertJson([
 			'id' => $this->albumID1,
 			'title' => self::ALBUM_TITLE_1,
-			'is_public' => true,
 			'thumb' => $this->generateExpectedThumbJson($this->photoID1), // photo 1 is thumb, because starred photo are always picked first
 			'photos' => [
 				$this->generateExpectedPhotoJson(self::SAMPLE_FILE_MONGOLIA_IMAGE, $this->photoID2, $this->albumID1), // photo 2 is alphabetically first
 				$this->generateExpectedPhotoJson(self::SAMPLE_FILE_TRAIN_IMAGE, $this->photoID1, $this->albumID1), // despite that photo 1 is starred
 			],
 		]);
+		$responseForAlbum->assertJsonMissing(['policies']);
```

Comment:
Why? (Question holds for _all_ identical lines below.)

Reply:
I am currently refactoring this.
`policies` were only returned when logged in. This is no longer the case and the code has already changed. :)